en/includes/guides/organization-management/share-applications.md

```diff
@@ -1,43 +1,81 @@
+Applications in the root organization can be shared with child organizations. Once an application is shared with an organization, organization users may log in to the applications.
 
-Follow the steps below to share your applications with organizations.
+You may share the following applications with organizations.
 
-1. Login to your organization (root) on the {{ product_name }} Console.
-2. Go to **Applications** and select the application that you want to share with your organizations.
-3. Go to the **Shared Access** tab and select **Share with only selected organizations**.
+- Registered applications - Any application that you register in {{product_name}}.
 
-    !!! note
-        Select **Share with all organizations** if you want to share your application with all existing organizations and any new organizations you may create in the future.
+- My Account portal - The self-service portal available for users of the root organization.
+
+## Share a registered application
+
+Follow the steps below to share a registered application with organizations.
+
+1. Login to the root organization on the {{ product_name }} Console.
+
+2. Go to **Applications** and select the application that you want to share with organizations.
+
+3. Go to its **Shared Access** tab and do one of the following:
 
-4. Select the organizations you wish to share your application with.
+    - Select **Share with all organizations** for the application to be shared with all existing organizations and new organizations you may create in the future.
 
-    ![Share application with organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/share-application.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+    -   Select **Share with only selected organizations** and select the relevant organizations.
 
-5. Click **Update** to save the configurations.
-6. Go to the **Protocol** tab of the application, be sure that the following [grant types]({{base_path}}/references/grant-types/) are allowed.
+        ![Share application with organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/share-application.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+4. Click **Update** to save the configurations.
+5. Go to the **Protocol** tab of the application and under **Allowed grant types**, ensure the following [grant types]({{base_path}}/references/grant-types/) are selected.
 
     - Client Credential
     - Code
     - Organization Switch
 
     !!! note
-        The `Organization Switch` grant will only be visible after you share the application with organizations.
 
-7. Click **Update** to save the changes.
+        The `Organization Switch` grant will only be visible after you have shared the application with one or more organizations.
+
+6. Click **Update** to save the changes.
+
+## Share the My Account Portal
```

Comment:
In IS7.0.0 Myaccount is shared by default. If required admin can unshare/ shared with set of orgs.

Let's change it for IS7 docs

Reply:
updated content for IS7

<img width="1056" alt="image" src="https://github.com/wso2/docs-is/assets/31583807/76f45de8-c1c9-4674-9eac-ca4cbef92581">